log/handler.go

```diff
@@ -57,7 +57,7 @@ type TerminalHandler struct {
 //
 // Example:
 //
-//	[DBUG] [May 16 20:58:45] remove route ns=haproxy addr=127.0.0.1:50002
```

Comment:
This was correct, it's the way the logger outputs it

Reply:
I didn't know it. I restored it back :)